.rubocop.yml

```
@@ -6,7 +6,7 @@ inherit_from:
   - .rubocop_todo.yml
 
 AllCops:
-  TargetRubyVersion: 2.3
+  TargetRubyVersion: '2.7'
   TargetRailsVersion: 5.1
```

While you're updating:
```suggestion
  TargetRailsVersion: 6.1
```